src/OrchardCore.Modules/OrchardCore.Localization/ModularPoFileLocationProvider.cs

```diff
@@ -74,6 +74,9 @@ public IEnumerable<IFileInfo> GetLocations(string cultureName)
 
                 // \src\OrchardCore.Cms.Web\Localization\OrchardCore.Cms.Web-fr-CA.po
                 yield return _fileProvider.GetFileInfo(PathExtensions.Combine(_resourcesContainer, extension.Id + CultureDelimiter + poFileName));
+
+                // \Localization\[CultureName]\[ModuleId].po
+                yield return _fileProvider.GetFileInfo(PathExtensions.Combine(_resourcesContainer, cultureName, extension.Id + PoFileExtension));
```

Comment:
We do not support the format `/Localisation/[CultureName]/[ModuleId].po` as mentioned in the [documentation](https://docs.orchardcore.net/en/latest/reference/modules/Localize/).